lib/mqtt.ex

```diff
@@ -0,0 +1,79 @@
+defmodule Mqtt do
+  @moduledoc """
+  `Oc2Mqtt` is main module for handling openc2 over mqtt
+  """
+
+  require Logger
+
+  @doc """
+  Start initializes system variables
+  and starts supervisor of mqtt client
+  """
+  def start do
+    client_id =
+      System.get_env("CLIENT_ID") ||
+        raise """
+        environment variable CLIENT_ID is missing.
+        For example:
+        export CLIENT_ID=:sfractal2020
+        """
+
+    Logger.info("client_id is #{client_id}")
+
+    mqtt_host =
+      System.get_env("MQTT_HOST") ||
+        raise """
+        environment variable HOST is missing.
+        Examples:
+        export MQTT_HOST="34.86.117.113"
+        export MQTT_HOST="mqtt.sfractal.com"
+        """
+
+    Logger.info("mqtt_host is #{mqtt_host}")
+
+    mqtt_port =
+      String.to_integer(
+        System.get_env("MQTT_PORT") ||
+          raise("""
```

Comment:
I like how explicitly you have raised the errors with clear instructions on what to do 🎉